duckduckgo/Android
# Rename whitelist to allowlist

Task/Issue URL: https://app.asana.com/0/488551667048375/1205221243351454/f

### Description
Rename whitelist to allowlist

### Steps to test this PR
* Install an older version of the app
* Add a site to the list of unprotected ones
* Install this one
* Check app starts normally and the unprotected sites list is unchanged

### UI changes
No UI changes
